docs/notes/2.25.x.md

```diff
@@ -68,6 +68,41 @@ Also, the PBS "release tag" will be inferred for PBS releases supplied via the `
 
 The default version of the [Pex](https://docs.pex-tool.org/) tool has been updated from 2.20.3 to [2.27.1](https://github.com/pex-tool/pex/releases/tag/v2.24.3).  Among many improvements and bug fixes, this unlocks support for pip [24.3.1](https://pip.pypa.io/en/stable/news/#v24-3-1).
 
+##### Black
+
+[The `pants.backend.python.lint.black` backend](https://www.pantsbuild.org/2.25/reference/subsystems/black) now uses version 24.8.0 by default, upgrading from 23.12.1. This comes with a new stable style (see [release notes for 24.1.0](https://black.readthedocs.io/en/stable/change_log.html) for details) which may result in extensive reformatting.
+
+To override Pants' default version, use [the `install_from_resolve` option](https://www.pantsbuild.org/2.25/reference/subsystems/black#install_from_resolve) and/or [the `interpreter_constraints` option](https://www.pantsbuild.org/2.25/reference/subsystems/black#interpreter_constraints). The style changes may be extensive, so, to make upgrading to Pants 2.25 easier, we provide the old lockfile as built-in, for now (Pants may remove this in future, so you should either switch to your own lockfile, or upgrade to the default Black 24). To use this lockfile, and remain on Black 23.3, configure a resolve as follows:
```

Comment:
```suggestion
To override Pants' default version, use [the `install_from_resolve` option](https://www.pantsbuild.org/2.25/reference/subsystems/black#install_from_resolve) and/or [the `interpreter_constraints` option](https://www.pantsbuild.org/2.25/reference/subsystems/black#interpreter_constraints). The style changes may be extensive, so, to make upgrading to Pants 2.25 easier, we provide the old lockfile as built-in, for now (Pants remove this in a future release, so you should either switch to your own lockfile, or upgrade to the default Black 24). To use this lockfile, and remain on Black 23.3, configure a resolve as follows:
```

Reply:
At the limit eventually the CPython version compatible with the old black version will be EoL.  We can't support it forewever.

Reply:
Yeah, might as well be stronger, and then people are pleasantly surprised if we take a few years to remove it, vs. being unpleasantly surprised if we say "may" and end up having to remove it in 2.26.